assets/js/util/abilities.js

```diff
@@ -1,7 +1,7 @@
 import { store } from '../store/configureStore';
 
-const userCan = (action, itemType, item) => {
-  const user = store.getState().user
+const userCan = (action, itemType, item, user) => {
```

Comment:
Same issue as above, this is temporary. We could just not import the store into this file, and use mapStateToProps and pass the user into this function to link it to the store.